src/Database/PostgreSQL/PQTypes/ExtrasOptions.hs

```diff
@@ -1,16 +1,20 @@
-module Database.PostgreSQL.PQTypes.ExtrasOptions (
-  ExtrasOptions(..)
+module Database.PostgreSQL.PQTypes.ExtrasOptions
+  ( ExtrasOptions
+  , eoForceCommit
+  , eoEnforcePKs
```

Comment:
Why export fields separately?

Reply:
So that clients would be forced to use
```
defaultExtrasOptions { eoForceCommit = ...
                     , ... 
                     }
```
to construct `ExtrasOptions` values instead of
```
ExtrasOptions { eoForceCommit = ...
              , ... 
              }
```
which will break if we add additional `ExtrasOptions` fields. Also lets us add new fields to `ExtrasOptions` without bumping the major version.

Reply:
+1

That seems to be a common pattern, for example with `aeson`'s `Options` and `defaultOptions`:
https://hackage.haskell.org/package/aeson-1.4.3.0/docs/Data-Aeson.html#t:Options

Reply:
What if someone wants it to break as they are interested in knowing any new options that might be added in the future?

Reply:
@arybczak that's a good point, because adding options without bumping major version might actually be a _bad_ thing. It would constrain us to, and force us to be careful that the default for the newly added field does not affect the behaviour, as otherwise we might be silently breaking clients.

Reply:
Although there's a counterargument to that: if the change is significant, say hypothetically `eoForceCommit` is split into two separate options, then `eoForceCommit :: Bool -> ExtrasOptions` ceases to exist and becomes a major version bump.

Reply:
Normally users don't, but I can add an `.Internal` module.

Reply:
Let's not complicate things. Exporting the constructor is fine.

Reply:
OK, done.